components/dashboard/dashboard.go

```diff
@@ -40,6 +40,9 @@ var imageParamMap = map[string]string{
 	"odh-dashboard-image": "RELATED_IMAGE_ODH_DASHBOARD_IMAGE",
 }
 
+// Verifies that Dashboard implements ComponentInterface
+var _ components.ComponentInterface = (*Dashboard)(nil)
```

Comment:
hum, duplicated as line89?

Reply:
Good catch. I blame morning rebase for and GoLand's auto-merge magic for that :)